### PR TITLE
[1.3.3] Disable Spam

### PR DESCRIPTION
**Pull request by scafroglia93**
- This don't break compile on arm and arm64
- Boot fine on Z3 wait for test on arm64

### CODE REVIEW

File: kernel/audit.c [hunk 1]
```diff
@@ -373,7 +373,7 @@ static void audit_printk_skb(struct sk_buff *skb)
 
 	if (nlh->nlmsg_type != AUDIT_EOE) {
 		if (printk_ratelimit())
-			printk(KERN_NOTICE "type=%d %s\n", nlh->nlmsg_type, data);
+			pr_debug(KERN_NOTICE "type=%d %s\n", nlh->nlmsg_type, data);
```
Comment (kholk) [inline kernel/audit.c]: Wrong. Those messages are useful and they indicate that you need to fix stuff.

File: sound/soc/msm/qdsp6v2/q6afe.c [hunk 2]
```diff
@@ -980,7 +980,7 @@ static int afe_get_cal_topology_id(u16 port_id, u32 *topology_id)
 	struct audio_cal_info_afe_top   *afe_top_info = NULL;
 
 	if (this_afe.cal_data[AFE_TOPOLOGY_CAL] == NULL) {
-		pr_err("%s: [AFE_TOPOLOGY_CAL] not initialized\n", __func__);
+		pr_debug("%s: [AFE_TOPOLOGY_CAL] not initialized\n", __func__);
 		return -EINVAL;
 	}
```
Comment (kholk) [inline sound/soc/msm/qdsp6v2/q6afe.c:985]: Those REALLY are errors. Not debugging crap.

File: drivers/mmc/core/core.c [hunk 3]
```diff
@@ -2497,7 +2497,7 @@ int mmc_resume_bus(struct mmc_host *host)
 	if (!mmc_bus_needs_resume(host))
 		return -EINVAL;
 
-	printk("%s: Starting deferred resume\n", mmc_hostname(host));
+	pr_debug("%s: Starting deferred resume\n", mmc_hostname(host));
```
Comment (kholk) [inline drivers/mmc/core/core.c:2500]: Acceptable

**Comment by scafroglia93** [comment]
No problem, if there is to do a rebase tell me what you're interested in

**Comment by kholk** [comment]
Only the mmc commit is valid for me, pr it alone please

**Comment by scafroglia93** [comment]
Done

**Comment by scafroglia93** [comment]
@jerpelea This was approved

**Pull request merged**